framer/Material-UI.framerfx/code/IconButton.tsx

```diff
@@ -6,6 +6,7 @@ import { Icon } from './Icon';
 
 interface Props extends MuiIconButtonProps {
   disabled: boolean;
+  inclusiveDisabled: boolean;
```

Comment:
I agree with @mbrookes (https://github.com/mui-org/material-ui/pull/27719#issuecomment-898961223) about the name `inclusiveDisable`, it's a little strange. 

What do you think of `disabledClickOnly`? Beneftis:
- autocomplete will work when start typing `dis...` - more people will notice the new prop.
- It reflects the purpose: only the click is removed - all the other interactions are still possible.

Reply:
> autocomplete will work when start typing dis... - more people will notice the new prop.

That is a great point. 

> It reflects the purpose: only the click is removed - all the other interactions are still possible.

Yeah, I'm a bit split on this part. Because I generally don't want to name props after "how" they do something but rather what they're for. For example, I rather name a prop "error" instead of "red" even though "red" is what's actually happening.

Though I lean towards optimizing for having the prop right next to `disabled` when it comes to sorting. And "inclusive" is almost as bad as the generic "accessible" tag line: "inclusive" for whom?